Add nsf eager prod project for Ziwei

PanZiwei opened the pull request:
## Problem
NSF EAGER needs a project to run [vcf2maf](https://github.com/Sage-Bionetworks-Workflows/nf-vcf2maf) pipeline on Seqera prod platform

## Solution
Add nsf eager prod project for Ziwei

Commit:
Add NSF EAGER prod project

thomasyu888 reviewed (commented):
@zaro0508 this will most likely be blocked by the limitation of # of batch compute envs

@PanZiwei can you look into this PR?  There are more changes than expected, I'd expect only the addition into the "projects-prod" folder

Commit:
Update nsf-eager prod project file

Commit:
Update nsf-eager prod project file

PanZiwei commented:
@thomasyu888 Updated the PR. Just 1 changes - add `config/projects-prod/nsf-eager-prod-project.yaml`

zaro0508 committed:
[IT-3888] Add a test user for py-orca (#478)

The py-orca runs tests by trigging an example-dev-project workflow.
We need to add our service user to seqera tower so it will
have access to the tower project.

Note: This user is not in AWS, it's only in seqera tower. We need
to add it as an AWS ARN because the configure-tower-project.py script
takes users from the S3ReadWriteAccessArns and automatically
adds them to the seqera project.

zaro0508 committed:
[IT-3888] Add a test user for py-orca prod (#483)

This is the same as PR #478 to add a test user for tower-dev which worked
successfully.  Now we are adding the same test user for tower (prod)
environment.

zaro0508 committed:
[IT-2361] Update Seqera enterprise version (#477)

This is a 2nd attempt at updating to v25.2.  The first attempt
in #474 failed to deploy and was reverted in #476. Seqera
support says the fix is to update to the latest v25.2 version.

Update to Seqera enterprise to v25.2.3[1]. This is an enterprise
version that supports the AWS cloud compute environment[2]
which would allow getting past the 50 job queue per account
per region constraint.

[1] https://docs.seqera.io/changelog/seqera-enterprise/v25.2.3
[2] https://docs.seqera.io/platform-enterprise/compute-envs/aws-cloud

zaro0508 committed:
[IT-2361] Update Seqera enterprise version (#477)

This is a 2nd attempt at updating to v25.2.  The first attempt
in #474 failed to deploy and was reverted in #476. Seqera
support says the fix is to update to the latest v25.2 version.

Update to Seqera enterprise to v25.2.3[1]. This is an enterprise
version that supports the AWS cloud compute environment[2]
which would allow getting past the 50 job queue per account
per region constraint.

[1] https://docs.seqera.io/changelog/seqera-enterprise/v25.2.3
[2] https://docs.seqera.io/platform-enterprise/compute-envs/aws-cloud

zaro0508 committed:
Revert "[IT-2361] Update Seqera enterprise version (#477)" (#484)

This reverts commit 9c54277d2cbdd67b108e7ea468f28c03c9c0feb7.

Deployment failed with a bunch of "No PropertySource found for file name"
errors.

zaro0508 committed:
Revert "[IT-2361] Update Seqera enterprise version (#477)" (#484)

This reverts commit 9c54277d2cbdd67b108e7ea468f28c03c9c0feb7.

Deployment failed with a bunch of "No PropertySource found for file name"
errors.

Commit:
Update nsf-eager prod project
